ui/app/AppLayouts/Wallet/popups/AddEditSavedAddressPopup.qml

```diff
@@ -76,38 +77,35 @@ StatusModal {
         }
 
         // To-Do use StatusInput within the below component
-        Item {
-            width: parent.width
-            height: addressInput.height
-            RecipientSelector {
-                id: addressInput
-                anchors.left: parent.left
-                anchors.leftMargin: Style.current.padding
-                anchors.top: parent.top
-                width: parent.width
-                accounts: RootStore.accounts
-                contactsStore: root.contactsStore
-                label: qsTr("Address")
-                input.placeholderText: qsTr("Enter ENS Name or Ethereum Address")
-                labelFont.pixelSize: 15
-                labelFont.weight: Font.Normal
-                input.implicitHeight: 56
-                isSelectorVisible: false
-                addContactEnabled: false
-                onSelectedRecipientChanged: {
-                    root.address = selectedRecipient.address
-                }
-                readOnly: root.edit
-                wrongInputValidationError: qsTr("Please enter a valid ENS name OR Ethereum Address")
+        RecipientSelector {
+            id: addressInput
+            implicitWidth: parent.width
+            inputWidth: implicitWidth
+            accounts: RootStore.accounts
+            contactsStore: root.contactsStore
+            label: qsTr("Address")
+            input.placeholderText: qsTr("Enter ENS Name or Ethereum Address")
+            labelFont.pixelSize: 15
+            labelFont.weight: Font.Normal
+            input.implicitHeight: 56
+            input.textField.anchors.rightMargin: 0
+            isSelectorVisible: false
+            addContactEnabled: false
+            onSelectedRecipientChanged: {
+                root.address = selectedRecipient.address
             }
+            readOnly: root.edit
+            wrongInputValidationError: qsTr("Please enter a valid ENS name OR Ethereum Address")
         }
     }
 
-    rightButtons: [
-        StatusButton {
-            text: root.edit ? qsTr("Save") : qsTr("Add address")
-            enabled: _internal.valid && _internal.dirty
-            onClicked: save(name, address)
+    footer: StatusDialogFooter {
+        rightButtons:  ObjectModel {
+            StatusButton {
+                text: root.edit ? qsTr("Save") : qsTr("Add address")
+                enabled: _internal.valid && _internal.dirty
+                onClicked: save(name, address)
```

Comment:
this should read `root.save(name, address)`, different context

Reply:
done!